.htaccess

```diff
@@ -77,6 +77,8 @@ Redirect 301 "/zh/blog/2021/11/30/use-apisix-ingress-in-kubesphere/httpbin.org/"
 Redirect 301 "/zh/blog/2021/08/25/Auth-with-Casbin-in-Apache-APISIX/" "/zh/blog/2021/08/18/auth-with-casbin-in-apache-apisix/"
 Redirect 301 "/zh/blog/2022/03/02/apisix-integration-graphql-plugin/" "/zh/blog/2022/03/02/apisix-integration-graphql/"
 
+RedirectMatch 301 "^/docs/ingress-controller/next/(.*)$" "/docs/ingress-controller/1.8.0/$1"
```

Comment:
`next` directory should point to `2.x` I think?

Reply:
Hi @juzhiyuan, please check this when you have time.